zetaclient/testutils/testdata.go

```diff
@@ -24,19 +24,6 @@ const (
 	RestrictedBtcAddressTest = "bcrt1qzp4gt6fc7zkds09kfzaf9ln9c5rvrzxmy6qmpp"
 )
 
-// SaveObjectToJSONFile saves an object to a file in JSON format
```

Comment:
Can we keep `SaveObjectToJSONFile`? It's been used to save downloaded JSON-based test data.

Reply:
Ok, I will add back and add comment for context

Reply:
6e57c8f9